features/step_definitions/web_steps.rb

```diff
@@ -65,10 +65,11 @@
 end
 
 When /I repeatedly answer questions/ do
```

Comment:
This wasn't working on my computer, but maybe it's my setup that's weird (I couldn't get the test in general to work);
potentially wrap this in a `while page.has_button?('submit-response-btn') do `, or recursively call `step "I repeatedly answer questions"` while the button exists, in order to future-proof it against multiple questions?

Reply:
Let me try doing the while loop 👍